openfecli/tests/commands/test_gather.py

```diff
@@ -237,7 +243,7 @@ def rbfe_results():
 
 
 @pytest.mark.download
-@pytest.mark.xfail
+@pytest.mark.xfail  # these results are outdated
```

Comment:
I would be fine with deleting the pooch test for now since it's outdated, with the expectation that we soon implement all our large data being downloaded with pooch or similar.

Reply:
Just remove for now imho, no need to keep things around if they don't help.